src/api/app/assets/javascripts/webui/comment.js

```diff
@@ -45,7 +45,13 @@ $(document).ready(function(){
 
   // This is being used to render a new root comment by the beta request show view
   $('.comment_new').on('ajax:complete', '.post-comment-form', function(_, data) {
-    $(this).closest('.comment_new').prev().append(data.responseText);
+    $(this).closest('.comment_new').prev().append(
+      '<div class="timeline-item">' +
```

Comment:
Without this dirty hack, new comments wouldn't render properly since they're missing those two `div`s. To properly address this issue, we need to redo how we create comments in `Webui::CommentsController`. New root comments are handled the same way as replies.